great_expectations/data_context/store/store_backend.py

```diff
@@ -4,3 +4,5 @@
 from great_expectations.data_context.store.in_memory_store_backend import (
     InMemoryStoreBackend,
 )
+
+__all__ = ["StoreBackend", "InMemoryStoreBackend"]
```

Comment:
What's the purpose of this? We're defining the `import *` behavior?

Reply:
Yes, and it resolves an "import unused" flake8 warning